experimental/navigation/types/navigation.ts

```diff
@@ -0,0 +1,18 @@
+interface TNavigationContainer {
```

Comment:
yeah, I saw in the past that people used `T` as prefix when describing a shape for typescript (but declaring that as a `type`). I think in here it would make sense to update the name in favor of the `I` prefix since this is declared using `interface` instead